### ISSUE TITLE
Document how the code is generated from the OpenAPI spec

### ISSUE THREAD

**Issue by castroal** [openai/openai-dotnet#201]
### Confirm this is not a feature request for the underlying OpenAI API.

- [X] This is not a feature request for the underlying OpenAI API

### Confirm this is not a feature request for Azure OpenAI.

- [X] This is not a feature request for Azure OpenAI

### Describe the feature or improvement you're requesting

In the README, it is stated that the client is generated from the OpenAPI specification. Please document the process for doing this.

**Comment by HavenDV** [comment]
Yes, I would also be interested to know about this. It's a different approach than Kiota and others I know of.

We also maintain our own fully generated OpenAI SDK for LangChain.NET (https://github.com/tryAGI/OpenAI) with a series of extensions for convenience, and are not yet sure about switching to this if there are delays in updates (which we don't have because it is updated based on GitHub Actions automatically in most cases if there are no breaking changes)

**Comment by yankarinRG** [comment]
It looks like the client is generated from the OpenAPI spec using [AutoRest](https://github.com/Azure/autorest) ([doc](https://github.com/Azure/autorest/blob/main/docs/readme.md))

**Comment by jsquire** [comment]
Information on running code generation has been added to the contributing guide.
